src/components/CreateUser.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div>
+    <md-card style="padding : 10px;">
+      <md-card-header>
+        <h1 class="md-title">Create new account</h1>
+      </md-card-header>
+
+      <md-field :class="usernameClass">
+        <md-icon>person</md-icon>
+        <label>Username</label>
+        <md-input v-model="username" type="text" v-on:keyup="fromIncorrecUsernameToCorrect()" v-on:blur="validateUsername()" required/>
+        <span class="md-error">A username must contain at least five characters</span>
+      </md-field>
+
+      <md-field :class="emailClass">
+        <md-icon>email</md-icon>
+        <label>Email</label>
+        <md-input v-model="email" type="email" v-on:keyup="fromIncorrecEmailToCorrect()" v-on:blur="validateEmail()" required/>
+        <span class="md-error">Please provide valid email address</span>
+      </md-field>
+
+      <md-field :class="passwordClass">
+        <md-icon>lock</md-icon>
+        <label>Password</label>
+        <md-input v-model="password" type="password" v-on:keyup="fromIncorrecPasswordToCorrect" v-on:blur="validatePassword()" required/>
```

Comment:
W emailu i username w keyup jest wywolanie metody z (), za to tu jest tylko nazwa

Reply:
poprawione